### DISCUSSION TITLE
Nuxt 3 Timeline

### DISCUSSION THREAD

**Discussion by djschilling** [nuxt/nuxt#18017] [Questions] [answered]
Is there somewhere a current timeline of the nuxt 3 Beta.

Most interesting date ist when the stable release will be.

I think i found a post that said that the stable release will be December 2021. Is there anywhere an update on that?

**Comment by wobsoriano** [comment, marked as answer]
https://github.com/nuxt/framework/discussions/2198#discussioncomment-1951587
